pkg/backup/backupCreateManager.go

```diff
@@ -41,6 +43,19 @@ func (bcm *backupCreateManager) create(ctx context.Context, backup *v1.Backup) e
 		return fmt.Errorf("failed to set status [%s] in backup resource: %w", v1.BackupStatusInProgress, err)
 	}
 
+	backup.Status.StartTimestamp = metav1.Now()
+	backup, err = bcm.client.UpdateStatus(ctx, backup, metav1.UpdateOptions{})
```

Comment:
Does this UpdateStatus maybe also need a `retryOnConflict`? 
The other UpdateStatus-methods have that

Reply:
If we always use the return value of the previous update function, conflicts are pretty unlikely and can only be caused by changing the resource externally (or from another process - with just the right timing). In the `defer` for setting the `CompletionTimestamp`, we cannot use the return value of the previous update, so we'll almost certainly have to resolve conflicts.

So, if we wanted to be absolutely safe, we could use `retryOnConflict`.